### PR TITLE
Update week 10 dg

### PR DESCRIPTION
**Pull request by ChongCheeKaiClarence**
No diagrams yet.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -378,18 +378,58 @@ The following activity diagram summarizes what happens when a user executes a ta
...
+
+The following activity diagram summarizes what happens when a user executes a tag add command:
+
+(insert activity diagram here)
+
+#### Design consideration
+
+**Aspect: How to implement tag edit:**
+
+* **Alternative 1 (current choice):** Creates a new tag and replace the old tag with the new one.
```
Comment (florentianayuwono) [inline docs/DeveloperGuide.md]:
```suggestion
* **Alternative 1 (current choice):** Creates a new tag and replaces the old tag with the new one.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -378,18 +378,58 @@ The following activity diagram summarizes what happens when a user executes a ta
...
+    * Pros: Prevents direct access into the information of a tag.
+    * Cons: Tedious. Necessary to manually change the old tag in every person and every task.
+
+* **Alternative 2:** Change the tag's name .
```
Comment (florentianayuwono) [inline docs/DeveloperGuide.md]:
```suggestion
* **Alternative 2:** Change the tag's name.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -378,18 +378,58 @@ The following activity diagram summarizes what happens when a user executes a ta
...
+    * Pros: Easy to implement.
+    * Cons: Easy to access into a tag. Could cause accidental bugs.
```
Comment (florentianayuwono) [inline docs/DeveloperGuide.md]:
```suggestion
    * Cons: Can potentially introduce bugs due to direct access into the tag's implementation.
```

**Review by florentianayuwono — CHANGES_REQUESTED** [review]
Great job with the documentation! Some suggestions here :D

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (ChongCheeKaiClarence) [reply docs/DeveloperGuide.md]: Changes made.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ChongCheeKaiClarence) [reply docs/DeveloperGuide.md]: Done!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ChongCheeKaiClarence) [reply docs/DeveloperGuide.md]: Ok. I'll make the changes.

**Comment by florentianayuwono** [comment]
LGTM! idk how to approve suddenly in the review tab there's only submit review not showing the approve one haha

**Pull request merged**